bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TopPhilicZp/ttzp/ttzp_proc_card.dat

```diff
@@ -0,0 +1,53 @@
+set group_subprocesses Auto 
+
+set ignore_six_quark_processes False 
+
+set loop_optimized_output True 
+
+set complex_mass_scheme False 
+
+import model Top-Philic_UFO_V1 
+
+define p = p b b~ 
+
+define l+ = l+ ta+ 
+
+define l- = l- ta- 
+
+generate p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > j j) , (t~ > b~ w-, w- > j j) 
+
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > j j) , (t~ > b~ w-, w- > j j) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~)), (t > b w+, w+ > j j) , (t~ > b~ w-, w- > j j) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~) 
+
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > l- vl~)), (t > b w+, w+ > j j) , (t~ > b~ w-, w- > j j) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~)), (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > l- vl~) 
+
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > j j) , (t~ > b~ w-, w- > l- vl~)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > l- vl~) 
+
+add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j) 
```

Comment:
line 44 is identical with line 31, line 44 should be "add process p p > t t~ v1 , (v1 > t t~ , (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > j j)), (t > b w+, w+ > l+ vl) , (t~ > b~ w-, w- > l- vl~) "